sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3103,14 +3103,24 @@ object GpuOverrides extends Logging {
       }),
     expr[MapConcat](
       "Returns the union of all the given maps",
+      // Currently, GpuMapConcat supports nested values but not nested keys.
+      // We will add the nested key support after
+      // https://github.com/rapidsai/cudf/pull/9452 is merged.
```

Comment:
I don't think this is the relevant cudf PR. The dependency should be https://github.com/rapidsai/cudf/issues/11093.

Reply:
Updated!